config/recipes/beats/filebeat_openshift.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: filebeat
+  namespace: openshift-monitoring
```

Comment:
`openshift-*` and `kube-*` projects are reserved for Red Hat. We should not deploy the components here. You can find more information on the page [Support considerations for monitoring](https://docs.openshift.com/container-platform/4.6/monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack).

Reply:
Excluded any reference of the `openshift-monitoring` namespace